capsules/src/spi.rs

```diff
@@ -35,6 +47,15 @@ pub struct Spi<'a, S: SpiMasterDevice + 'a> {
     kernel_len: Cell<usize>,
 }
 
+pub struct SpiSlave<'a, S: SpiSlaveDevice + 'a> {
+    spi_slave: &'a S,
+    busy: Cell<bool>,
+    app: MapCell<SlaveApp>,
```

Comment:
Oh, interesting. Should we then also change the regular SPI master implementation (just a few lines above it) that also uses a `MapCell`?